### PR TITLE
ericscript conversion

### PR DESCRIPTION
**Pull request by hiraksarkar**
Added the ericscript converter to the new branch, one problem is for few lines the gene position is not available and therefore I have put a -1 there to signify it is unavailable. Let me know if there is a better way to handle this. I kept the generic form of converter proposed by @wilcas .

### CODE REVIEW

File: ericscript_to_bedpe.py [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+        out_row['end2'] = -1
+
+    out_row['strand2'] = strand1
+
+
+    gene1 = input_row['GeneName1']
+    gene2 = input_row['GeneName2']
+    out_row['name'] = '{G1}-{G2}'.format(G1=gene1, G2=gene2)
+
+    out_row['score'] = input_row['EricScore']
```
Comment (wilcas) [inline ericscript_to_bedpe.py:56]: Note that EricScore is an aggregate  of other scores employed by the author, which are reported in the output. See paper: [Discovering chimeric transcripts in paired-end RNA-seq data by using EricScript](https://academic.oup.com/bioinformatics/article/28/24/3232/247436/Discovering-chimeric-transcripts-in-paired-end-RNA)

Comment (hiraksarkar) [reply ericscript_to_bedpe.py:56]: Should I drop this and store a 0?

File: civic_to_bedpe.py [hunk 2]
```diff
@@ -0,0 +1,126 @@
+
+
+'''
+Convert EricScript output to BEDPE format.
```
Comment (wilcas) [inline civic_to_bedpe.py]: `Convert CIViC database format to BEDPE format`

File: civic_to_bedpe.py [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+    out_row = dict()
+
+    [chr1, start1, end1] = input_row['chromosome'], input_row['start'], input_row['stop']
+
+    out_row['chrom1'] = 'chr{num}'.format(num=chr1)
+    if(start1.isdigit() and end1.isdigit()):
+        out_row['start1'] = int(start1)
+        out_row['end1'] = int(end1)
+    else:
+        out_row['start1'] = '.'
```
Comment (wilcas) [inline civic_to_bedpe.py]: BEDPE specifies that unknown locations should be specified as `-1`.  Change lines 34, 35, 47, and 48 to reflect this

File: civic_to_bedpe.py (hunk 2 above)
Comment (hiraksarkar) [reply civic_to_bedpe.py]: oops forgot to change the comments

File: civic_to_bedpe.py (hunk 3 above)
Comment (hiraksarkar) [reply civic_to_bedpe.py]: fixing ! thanks.

File: civic_to_bedpe.py [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+        if (heading not in out_row) and (heading in input_row):
+            out_row[heading] = input_row[heading]
+        elif heading not in out_row:
+            out_row[heading] = '.'
+    return out_row
+
+
+def add_fields(bedpe_fields):
+    '''Add fields from input to end of BEDPE format'''
+    '''Specific to EricScript'''
```
Comment (wilcas) [inline civic_to_bedpe.py]:
```
'''
Add fields from input to end of BEDPE format; Specific to CIViC
'''
```

**Review by wilcas — CHANGES_REQUESTED** [review]
BEDPE specifies that unknown positions should be specified as -1. Other than that it's just some minor things with format and naming

File: civic_to_bedpe.py (hunk 2 above)
Comment (hiraksarkar) [reply civic_to_bedpe.py]: done

File: civic_to_bedpe.py (hunk 3 above)
Comment (hiraksarkar) [reply civic_to_bedpe.py]: done

File: civic_to_bedpe.py (hunk 4 above)
Comment (hiraksarkar) [reply civic_to_bedpe.py]: done

File: ericscript_to_bedpe.py (hunk 1 above)
Comment (hiraksarkar) [reply ericscript_to_bedpe.py:56]: @rdocking any suggestion?

Comment (rdocking) [reply ericscript_to_bedpe.py:56]: No strong preference here

**Comment by rdocking** [comment]
Going to close this one, since the changes are also in the branch at https://github.com/rdocking/fusebench/pull/45

**Pull request closed without merging**